docs/api-status.json

```diff
@@ -1834,6 +1834,14 @@
       {
         "name": "API.RemoveUserCap",
         "comment": "RemoveUserCap removes the capabilities from a user.\nOn Success, it returns the updated list of UserCaps for the user.\n"
+      },
+      {
+        "name": "API.LinkBucket",
+        "comment": "LinkBucket - https://docs.ceph.com/en/latest/radosgw/adminops/#link-bucket\n"
+      },
+      {
+        "name": "API.UnlinkBucket",
+        "comment": "UnlinkBucket - https://docs.ceph.com/en/latest/radosgw/adminops/#unlink-bucket\n"
```

Comment:
This is still in the wrong section (should be in "preview", not in "stable"). Please remove the file and recreate with `make api-update`.